app/redux/actions/async.js

```diff
@@ -237,8 +237,9 @@ export function login(api, credentials, options, postLoginAction) {
             handleLoginFailure(ErrorMessages.ERR_FETCHING_USER, err);
           } else {
             const userHasClinicProfile = !!_.get(user, ['profile', 'clinic'], false);
+            const isClinicianAccount = personUtils.isClinicianAccount(user);
 
-            if (config.CLINICS_ENABLED) {
+            if (config.CLINICS_ENABLED && isClinicianAccount) {
```

Comment:
If we gate this login redirection path on `isClinicianAccount`, then won't that prevent non-clinic (classic or new) users who receive a clinician invite from a clinic from getting to the clinician workflow?

Reply:
Hmm. I just tested it with an invite using a new email address and it did in fact land the new clinician on the clinicDetails page as desired. Now, this is because I chose a 'Clinician' Account from the user sign up flow, and thus the new user had the clinic role.

Now, if I had chosen to create a standard patient account, or had invited via email an existing standard patient account, you're correct that they would not see the invitation. Should we allow invites to non-clinician accounts?

Either way, I suppose I'm not gating the corresponding route checks on refresh in routes.js, so perhaps I should just remove this gate here. My intention was to not bother with additional server calls if it wasn't a clinician user, but it's rather insignificant and harmless to leave them in. Any other thoughts on this?